tests/doc/tst_doc_images.py

```diff
@@ -145,7 +145,11 @@ def test_docs(test_case: _TestCaseTuple):
         _save_failed_test_image(fail_source)
         pytest.fail(fail_msg)
 
-    warn_msg, fail_msg = _test_compare_images(*test_case)
+    try:
+        warn_msg, fail_msg = _test_compare_images(*test_case)
```

Comment:
To me, it would better to have the try block within `_test_compare_images`, but it is a matter of taste.

Reply:
Ok, I also think is cleaner